flink-core/src/test/java/org/apache/flink/api/common/io/DelimitedInputFormatTest.java

```diff
@@ -200,6 +200,30 @@ public void testReadCustomDelimiter() {
 		}
 	}
 
+	@Test
+	public void testReadCustomDelimiterWithCharset() throws IOException {
```

Comment:
This tests the `DelimitedInputFormat` with default charset. 
Can you add a test with a different charset as well?